dirAlias.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     '#lib': resolvePath('src/app/lib/'),
     '#models': resolvePath('src/app/models/'),
     '#testHelpers': resolvePath('src/testHelpers/'),
+    '#utilities': resolvePath('src/server/utilities/'),
```

Comment:
This should be clear its server utilities, if someone imports `#utilities/serviceConfigs` it will break the client app

Reply:
I suggest we go with `#server` and use `#server/utilities`

Reply:
Makes sense